OWNERS

```diff
@@ -5,5 +5,12 @@ approvers:
 - justaugustus
 - mikedanese
 
+reviewers:
```

Comment:
Please add all the approvers to reviewers. Otherwise prow will only ever assign Verolop.